plugins/modem-manager/fu-mm-device.c

```diff
@@ -897,6 +1077,13 @@ fu_mm_device_udev_add_port (FuMmDevice	*self,
 		return;
 	}
 
+	if (g_str_equal (subsystem, "wwan") &&
```

Comment:
You're limiting the MBIM QDU download protocol exclusively to PCI devices now; is that expected? Is it possible to use this protocol through USB as well? Maybe we should also handle QMI ports in the wwan subsystem and MBIM ports in the usbmisc subsystem?